code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -234,6 +234,7 @@
 	throw_range = 3
 	hitsound = 'sound/items/trayhit1.ogg'
 	hit_reaction_chance = 50
+	materials = list(MAT_METAL = 500)
```

Comment:
A chair costs one sheet. The var on metal sheets reads as follows:
`materials = list(MAT_METAL=MINERAL_MATERIAL_AMOUNT)`